utils/get_idle_gpus.py

```diff
@@ -24,37 +24,29 @@
 assert len(sys.argv) == 2
 nb_idle_gpus = int(sys.argv[1])
 
-# dump the output of "nvidia-smi" command to file
-dump_file = './nvidia-smi-dump'
-with open(dump_file, 'w') as o_file:
-  subprocess.call(['nvidia-smi'], stdout=o_file)
+# assume: idle gpu has no more than 50% of total card memory used
+maxGpuUsageThreshold = .5
```

Comment:
Inconsistent variable naming convention.
Some variables are named as `aaa_bbb_ccc` (*e.g.* `nb_idle_gpus`) and some are named as `aaaBbbCcc` (*e.g.* `maxGpuUsageThreshold`).